Library/Homebrew/unpack_strategy/tar.rb

```diff
@@ -38,8 +39,9 @@ def extract_to_dir(unpack_dir, basename:, verbose:)
         end
 
         system_command! "tar",
-                        args:    ["xf", tar_path, "-C", unpack_dir],
-                        verbose: verbose
+                        args:         ["xf", tar_path, "-C", unpack_dir],
+                        extra_params: extra_params ||= [],
```

Comment:
I don't believe `system_command!` takes an `extra_params` named argument. This line can be deleted. Try…
```ruby
args:         ["xf", tar_path, "-C", unpack_dir, *extra_params],
```